electroncash_gui/qt/address_list.py

```diff
@@ -337,24 +338,28 @@ def doCopy(txt):
             addr_URL = web.BE_URL(self.config, 'addr', addr)
             if addr_URL:
                 menu.addAction(_("View on block explorer"), lambda: webopen(addr_URL))
-        else:
+            menu.addAction(
+                "Consolidate coins for address",
+                lambda: self._open_consolidate_coins_dialog(addr),
+            )
+
+        elif col > -1:
```

Comment:
Note for other reviewers: this is just a change in indentation of the following block. No other code change.